src/thunderbots/software/test/world/world.cpp

```diff
@@ -51,6 +51,26 @@ TEST_F(WorldTest, construction_with_parameters)
     EXPECT_EQ(ball, world.ball());
     EXPECT_EQ(friendly_team, world.friendlyTeam());
     EXPECT_EQ(enemy_team, world.enemyTeam());
+
+    // Test logic for world calculations
+    EXPECT_EQ(false, world.ballInFriendlyDefenseArea());
+    EXPECT_EQ(false, world.ballInEnemyDefenseArea());
+    Ball ballfriendly = Ball(Point(-4, 0.5), Vector(-0.3, 0), current_time);
```

Comment:
Everything at this line and below should probably be in it's own test. Each unit test should test one specific thing. This test is for the constructor, but the tests for `ballInFriendlyDefenseArea` should have their own (probably multiple) tests.

Reply:
OK done

Reply:
It looks like we're still testing multiple things in one test? Each test should only encompass a single thing, ie:
- `ball_in_friendly_defense_area`
- `ball_in_enemy_defense_area`
- `ball_just_outside_enemy_defense_area`
- etc.

This makes things much easier to debug if issues arise later.